2020-03-14 13:54:23 - root - ERROR -11_traceback模块:  Traceback (most recent call last):
  File "/Users/sylar/PycharmProjects/python课程录制/python_1_4/11_traceback模块.py", line 22, in <module>
    print(1/0)
ZeroDivisionError: division by zero

